code-of-conduct/code-of-conduct-en.md

```diff
@@ -12,22 +12,22 @@ The Code of Conduct aims to provide a positive, inspiring and welcoming communit
 
 This Code of Conduct applies to all people contributing to the Project, namely by online contributions, whether it is repositories, forum posts or on social media. 
 
-It also applies to public spaces, during PrestaShop event, namely the PSD, PS Connect, meet-up or any other event during which an individual or a group of individuals represent the Project. 
+It also applies to public spaces, during PrestaShop event, namely the PSD, PS Connect, meet-up or any other event during which an individual or a group of individuals represents the Project. 
 
 The Code of Conduct applies to any member of the PrestaShop community, whether it is PrestaShop employees, partners, ambassadors, moderators or any other members on the contributory sites and PrestaShop social media. 
 
 ## Expected behaviours 
 
-The Code of conduct aiming to create a positive and benevolent community, the following principles must be ensured in order to have constructive and respectful exchanges : 
+The Code of conduct aims to create a positive and benevolent community, the following principles must be ensured in order to have constructive and respectful exchanges : 
```

Comment:
```suggestion
The Code of conduct aims to create a positive and benevolent community, the following principles must be ensured in order to have constructive and respectful exchanges: 
```

Reply:
_aims at creating_ plutôt, non ?

Reply:
I wonder if "aim at" is not more about visually targetting something...